### PR TITLE
Lobby/Main Room component and other room components

### PR DESCRIPTION
**Pull request by parky22**

### CODE REVIEW

File: browser/react/components/App.js [hunk 1]
```diff
@@ -3,12 +3,14 @@ import '../../aframeComponents/scene-load';
...
 
+const style = { 'width': '100%', 'height': '100%' };
+
 export default function App (props) {
   return (
     // AssetLoader is a stateless component containing the a-assets for all of the React components
     //   rendered via props.children. It must reside here because A-Frame requires a-assets to a
     //   direct child of a-scene.
-    <div style={{ width: '100%', height: '100%' }}>
+    <div style={style}>
```
Comment (bushidocodes) [inline browser/react/components/App.js:13]: Nice optimization! 🥇

File: browser/react/components/AssetLoader.js [hunk 2]
```diff
@@ -2,16 +2,20 @@ import React from 'react';
...
       <a-asset-item id="tape-drive" src="/sean/tape-reader.dae"></a-asset-item>
       <a-asset-item id="disk-pack" src="/sean/diskpack.dae"></a-asset-item>
       <a-asset-item id="punch-reader" src="/sean/card-reader.dae"></a-asset-item>
+
+      {/* Lobby assets */}
+      <img id="slide" src="/img/class_copy.png"/>
+      <img id="podium" src="/img/fullstack.png"/>
+      <img id="floorText" src="/img/hardwood.jpg"/>
+      <a-mixin id="chair-part" geometry="primitive: box" material="color: brown"></a-mixin>
+      <a-asset-item id="monitor" src="/img/monitor/model.dae"></a-asset-item>
```
Comment (bushidocodes) [inline browser/react/components/AssetLoader.js]: ...and the devil frog was never heard from again (or so we thought).

File: browser/react/components/Joey.js [hunk 3]
```diff
@@ -14,9 +15,14 @@ export default class Joey extends React.Component {
   render () {
     return (
       <a-entity id="room" position="0 0 0">
+        <Room floorWidth="50"
+              floorHeight="50"
+              wallHeight="25"
+              wallColor="blue"
+              floorColor=""
```
Comment (bushidocodes) [inline browser/react/components/Joey.js:22]: part of me thinks that we should have a woodish brown color for "progressive enhancement" purposes. I don't think this needs to be done before merging. Just informational.

**Review by bushidocodes — APPROVED** [review]
Looks great Yoo!Nah!

File: browser/react/components/Chair.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export default (props) => (
+  <a-entity id="chair" position={`${props.x} ${props.y} ${props.z}`} rotation='0 180 0'>
```
Comment (bethqiang) [inline browser/react/components/Chair.js]: Change rotation to use double quotes for consistency?

File: browser/react/components/Lobby.js [hunk 5]
```diff
@@ -1,28 +1,68 @@
...
+        {/* Orbs */}
         <Teleporter x="-10" y="1" z="-1" color="red" href="/vr/sean" />
         <Teleporter x="-10" y="1" z="1" color="yellow" href="/vr/beth" />
         <Teleporter x="-10" y="1" z="3" color="blue" href="/vr/joey" />
         <Teleporter x="-10" y="1" z="5" color="purple" href="/vr/yoonah" />
+
+        {/* Chairs */}
+        {
+          createArray(10).map((el) => (
+            <Chair x={`${el[0]}`} y='0' z={`${-12.5 + el[1]}`} key={`${el[0] + ',' + el[1]}`} />
```
Comment (bethqiang) [inline browser/react/components/Lobby.js]: Double quotes here too? (And below)

File: browser/react/components/Yoonah.js [hunk 6]
```diff
@@ -15,9 +16,14 @@ export default class Yoonah extends React.Component {
   render () {
     return (
       <a-entity id="room" position="0 0 0">
```
Comment (bethqiang) [inline browser/react/components/Yoonah.js]: Not the first place this occurs probably but just noticed, do we need this `id`?

**Review by JoeyDarbyshire — APPROVED** [review]

**Review by bethqiang — APPROVED** [review]
Looks great to me! Some small things here and there but overall I agree with Joey, re: super clean & easy to read.

**Pull request merged**